templates/base.html

```diff
@@ -47,102 +48,114 @@
         {% block extrastyle %}{% endblock %}
     </head>
     <body>
-        <div class="off-canvas-wrapper">
-            <div class="off-canvas-wrapper-inner" data-off-canvas-wrapper>
-                <div class="off-canvas position-right" id="offCanvas" data-off-canvas data-position="right">
-                    <!-- Close button -->
-                    <button class="close-button" aria-label="Close menu" type="button" data-close>
-                        <span aria-hidden="true">&times;</span>
-                    </button>
+        {% if user.is_authenticated %}
```

Comment:
I was looking at a lot of other websites to see how they do it. For example: when you go to www.trello.com, if you are an unauth user you get an "about" section, but when you are logged in you get your trello boards. Similar with facebook and most other websites that have a login. 

How exactly do they do this? I think it would make sense for it to work in a similar way